lib/detect-testing-library-utils.ts

```diff
@@ -580,14 +583,22 @@ export function detectTestingLibraryUtils<
       return isRenderUtil(initIdentifierNode);
     };
 
-    const isDebugUtil: IsDebugUtilFn = (node) => {
-      return isTestingLibraryUtil(
-        node,
-        (identifierNodeName, originalNodeName) => {
-          return [identifierNodeName, originalNodeName]
-            .filter(Boolean)
-            .includes('debug');
-        }
+    const isDebugUtil: IsDebugUtilFn = (identifierNode, callExpressionNode) => {
```

Comment:
I'd avoid passing `callExpressionNode` as a second arg. Just use the member expression from `identifierNode.parent` and check it's defined and `isMemberExpression` to keep doing the same checks you are doing now.

Reply:
Updated here https://github.com/testing-library/eslint-plugin-testing-library/pull/349/commits/1778fa8ed6771e4869acbe0de254e1435cecab2a